test

```diff
@@ -26,12 +26,12 @@ $zigup default 0.5.0
 
 testCompilerCount 3
 
-touch scratch/install/0.6.0/keep
+$zigup keep 0.6.0
 $zigup clean # doesn't delete anything because we have keepfile and master doens't get deleted
 testCompilerCount 3
 
-rm scratch/install/0.6.0/keep
-$zigup clean 2>&1 | grep "because it is master" # DOESN't delete master because we dont delete master, but DOES delete 0.6.0
+$zigup clean 0.6.0
```

Comment:
we should verify the compiler count after running this clean command, would also be good to verify a message like "grep "deleting '0.6.0'"